TheMET/TheMET/FeaturedArtistsView/FeaturedArtistsView.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  FeaturedArtistsView.swift
+//  TheMET
+//
+//  Created by Анна Ситникова on 31/10/2023.
+//
+
+import Foundation
+import UIKit
+
+class FeaturedArtistsView: UIView {
```

Comment:
- This view is also tappable. Pls add a configuration on what to do on Tap. Use UITapGestureRecognizer to recognise the tap
- This view must be highlighted in some way when touched. Override the `UIView`'s methods `touchesBegan`, `touchesEnded`, `touchesCancelled` to detect the touch-events, highlight/de highlight when needed